go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/SAP/go-hdb v0.14.1
 	github.com/Sectorbob/mlab-ns2 v0.0.0-20171030222938-d3aa0c295a8a
 	github.com/aerospike/aerospike-client-go/v5 v5.6.0
-	github.com/aliyun/alibaba-cloud-sdk-go v0.0.0-20190620160927-9418d7b0cd0f
+	github.com/aliyun/alibaba-cloud-sdk-go v1.61.1311
```

Comment:
This should be held back.

Reply:
Yep, it's been reverted now by downgrading the deps on `github.com/hashicorp/vault-plugin-secrets-alicloud`